### ISSUE TITLE
Enhancement: Make Resources Dropdown in Nav Bar Keyboard Accessible

### ISSUE THREAD

**Issue by annakreno** [spacelabdev/spacelab-react#385]
- Currently, cannot tab into Resources dropdown in nav bar with a keyboard on desktop version of site, but need this for WCAG A compliance
- Add [ARIA states and properties](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes) to the div with class "nav-dropdown nav-link" so that keyboard-only users can expand the dropdown
